### DISCUSSION TITLE
Should docs include bit about adding link between fd and fd-find for Ubuntu and Debian systems.

### DISCUSSION THREAD

**Discussion by ZieMcd** [rest-nvim/rest.nvim#292] [General]
I am a real newbie when it comes to nvim, lua, and even Linux itself. When trying to get the telescope extension to work, none of the .env files were showing up. I copied the code from the extension, ran it, and got the "fd not found" error (I assume this would have been easier if I had run it in debug). Either way, running the command `ln -s $(which fdfind) ~/.local/bin/fd` fixed it. I'm not sure if anyone else has run into this issue, but I think it might be helpful to include a note in the docs.

**Comment by NTBBloodbath** [comment]
Addressed by #293, thank you! :)
